resourceDescriptors.yaml

```diff
@@ -687,11 +687,11 @@
     - name: pub
       url: https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&code=[%s]
 
-- db_prefix: OMIM
-  name: OMIM
-  aliases: ['omim']
-  example_id: OMIM:600483, OMIM:PS100200
-  gid_pattern: "^(OMIM|omim):[PS]*\\d+$"
+- db_prefix: MIM
+  name: MIM
+  aliases: ['mim']
```

Comment:
do you want to add an alias for "OMIM" as well here?

Reply:
@sierra-moxon You mean have:
`aliases: ['mim', 'omim']`
? It was omim and I changed it to mim. Adding 'omim' back is fine with me, I'm not clear on how that works.

Reply:
I don't think we make use of the aliases but might be good to have both of them.

Reply:
Looks good to me. I agree that having both MIM and OMIM as aliases would be good. 
Have we checked to make sure we aren't bringing in OMIM IDs from any other sources? I don't see them on the gene pages but I haven't checked all the loads.